x/government/keeper/grpc_query_test.go

```diff
@@ -25,24 +25,26 @@ func TestKeeper_GovernmentAddr(t *testing.T) {
 		{
 			"empty request",
 			nil,
-			governmentTestAddress,
+			nil,
 			true,
 		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			k, ctx := setupKeeperWithGovernmentAddress(t, tt.government)
 
+			require.Equal(t, tt.government, k.GetGovernmentAddress(ctx))
+
 			c := sdk.WrapSDKContext(ctx)
 
 			got, err := k.GovernmentAddr(c, tt.request)
 			if (err != nil) != tt.wantErr {
 				t.Errorf("Keeper.GovernmentAddr() error = %v, wantErr %v", err, tt.wantErr)
 				return
-			} else {
-				require.Equal(t, &types.QueryGovernmentAddrResponse{GovernmentAddress: tt.government.String()}, got)
 			}
-
+			if !tt.wantErr {
+				require.Equal(t, &types.QueryGovernmentAddrResponse{GovernmentAddress: k.GetGovernmentAddress(ctx).String()}, got)
```

Comment:
For me it's should be better
```golang
require.Equal(t, &types.QueryGovernmentAddrResponse{GovernmentAddress:tt.government.String()}, got)
```

Retrive from keeper the government address with `k.GetGovernmentAddress` and if it fails you will never reach `if !tt.wantErr` control. So get another time the government address from the keeper is unnecessary.